### ISSUE TITLE
[Regression]: EventEmitter.listeners("foo") throws if no listener exists

### ISSUE THREAD

**Issue by mxschmitt** [jfhbrook/pyee#103]
The following:

```py
from pyee import EventEmitter
ee = EventEmitter()
ee.listeners("foo")
```

results in:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/.../env/lib/python3.10/site-packages/pyee/base.py", line 229, in listeners
    return list(self._events[event].keys())
KeyError: 'foo'
>>> 
```

since version 9.0.0. In version `<8` it was returning an empty list instead. This most likely broke in #99 when the `defaultdict` of `EventEmitter._events` got changed to a normal `dict`.

**Comment by jfhbrook** [comment]
Yeah, sounds like a regression. I'll patch shortly.

**Comment by jfhbrook** [comment]
Released your patch as 9.0.1 - thanks so much!!
